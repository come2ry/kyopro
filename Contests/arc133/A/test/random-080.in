10
7 10 8 11 8 2 3 4 6 5
