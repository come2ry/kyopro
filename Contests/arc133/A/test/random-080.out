1 0 8 1 1 8 2 3 4 6 5
